### PR TITLE
Add "Help" menu

### PR DESCRIPTION
**Pull request by martinRenou**
Add "Help" menu with the "About WorkflowManager..." action which displays license informations

### CODE REVIEW

File: force_wfmanager/tests/test_wfmanager_task.py [hunk 1]
```diff
@@ -84,6 +85,10 @@ def mock_show_error(*args, **kwargs):
     return args
 
 
+def mock_info_dialog(*args, **kwargs):
```
Comment (stefanoborini) [inline force_wfmanager/tests/test_wfmanager_task.py]: i am growing excessively annoyed at the proliferation of these entities. We need to do something, but I am unsure about what.

File: force_wfmanager/wfmanager_task.py [hunk 2]
```diff
@@ -193,6 +203,15 @@ def open_workflow(self):
             else:
                 self.current_file = dialog.path
 
+    def open_about(self):
+        information(
+            None,
+            "Workflow manager: UI application for Business Decision System\n\n"
+            "It is part of the FORCE project\n"
```
Comment (stefanoborini) [inline force_wfmanager/wfmanager_task.py]: (Horizon 2020/NMBP-23-2016/721027).

File: force_wfmanager/wfmanager_task.py [hunk 3]
```diff
@@ -193,6 +203,15 @@ def open_workflow(self):
...
+            "This software is released under the BSD license",
+            "About WorflowManager"
```
Comment (stefanoborini) [inline force_wfmanager/wfmanager_task.py]: remove the above line

File: force_wfmanager/tests/test_wfmanager_task.py (hunk 1 above)
Comment (martinRenou) [inline force_wfmanager/tests/test_wfmanager_task.py]: Maybe put them in a separate file ?

File: force_wfmanager/wfmanager_task.py (hunk 3 above)
Comment (martinRenou) [inline force_wfmanager/wfmanager_task.py]: Maybe it is unclear, but "About WorkflowManager" is the name of the dialog. Do you me to remove this line ? The default title is "Information"

Comment (stefanoborini) [inline force_wfmanager/wfmanager_task.py]: ops, sorry! good as is. I didn't see the comma

**Pull request merged**